romancal/stpipe/core.py

```diff
@@ -4,6 +4,7 @@
 from stpipe import Step, Pipeline
 
 from .. import datamodels
+import roman_datamodels as rdatamodels
```

Comment:
we should agree on one alias `rdm` or `rdatamodels`.

Reply:
If I may vote, I strongly prefer rdm over rdatamodels (in fact, I'd prefer romandatamodels over the latter.. rdm is good for me either way).

Reply:
I vote rdm